lib/Rx/SchedulerInterface.php

```diff
@@ -17,4 +17,11 @@ public function schedule($action);
      * @return DisposableInterface
      */
     public function scheduleRecursive($action);
+
+    /**
+     * Gets the current representation of now for the scheduler
+     *
+     * @return mixed
```

Comment:
Should we make this a number instead and return `time()/microtime(true)` instead of a new `DateTime` object in the appropriate schedulers?